css/src/nc12.scss

```diff
@@ -72,3 +72,205 @@
 	box-shadow: unset;
 	filter: drop-shadow(0 1px 3px rgba(179, 179, 179, .75));
 }
+
+$color-main-text: #000;
+$color-main-background: #fff;
+$color-primary: #0082c9;
+$color-primary-text: #fff;
+
+@function nc-darken($color, $value) {
+	@return darken($color, $value);
+}
+
+@function nc-lighten($color, $value) {
+	@return lighten($color, $value);
+}
+
+$color-border: nc-darken($color-main-background, 8%);
+$border-radius: 3px;
+
+/* ---- jQuery UI datepicker ---- */
+.ui-widget.ui-datepicker {
+	margin-top: 10px;
+	padding: 4px 8px;
+	width: auto;
+	border-radius: $border-radius;
+	border: none;
+	background: unset;
+	background-color: $color-main-background;
+	filter: drop-shadow(0 1px 3px rgba(179, 179, 179, .75));
+
+	.ui-state-default,
+	.ui-widget-content .ui-state-default,
+	.ui-widget-header .ui-state-default {
+		border: 1px solid transparent;
+		background: inherit;
+	}
+
+	.ui-widget-header {
+		padding: 7px;
+		font-size: 13px;
+		border: none;
+		background-color: $color-main-background;
+		color: $color-main-text;
+
+		.ui-datepicker-title {
+			line-height: 1;
+			font-weight: 300;
+		}
+
+		.ui-icon {
+			opacity: .5;
+
+			&.ui-icon-circle-triangle-e {
+				background: url('../../../apps/tasks/css/../img/arrow-right.svg') center center no-repeat;
+			}
+
+			&.ui-icon-circle-triangle-w {
+				background: url('../../../apps/tasks/css/../img/arrow-left.svg') center center no-repeat;
```

Comment:
This looks like a compiled path. use the correct url and let the scss compiler rewritter do its job :)

Reply:
What would be the correct path? From the scss file to the svg file? Like `../img/arrow-left.svg`?

Reply:
Yes, relative to the task app directory :)